x/tally/types/abci_types.go

```diff
@@ -26,7 +26,7 @@ type Request struct {
 type RevealBody struct {
 	Salt         []byte   `json:"salt"`
```

Comment:
Maybe I'm nitting but it should be `[32]byte` or however golang does specific-sized arrays.

Reply:
Updated and added a custom JSON unmarshaller that gives an error if the salt in hex is not 32-byte long.